setup.py

```diff
@@ -49,6 +49,13 @@ def read(*rnames):
       package_dir={'': 'src'},
       install_requires=[
           'setuptools',
+
+      ],
+      setup_requires=[
+          'pytest-runner',
```

Comment:
Could this be moved to `test_requires`. Why is `pytest-runner` needed to run `setup.py` ?